### DISCUSSION TITLE
forge test -m "testSingleFun" not working.

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#1491] [Q&A] [answered]
Hi there, I'm trying to test single function running 'forge test -m testXyzFun' following Patrick collins in Lesson 7 Running tests on chain forks but it is throwing error in my terminal I'm using wsl terminal. How to solve this issue.

**Comment by EngrPips** [comment]
Hello @panditdhamdhere , `-m` is deprecated use `--mt` instead and the test would run

**Comment by ArnaudBand** [comment]
Hello @panditdhamdhere, I got the same error this how I fix it. I use `forge test --help``` to check which match to test a single test and I used `forge test --match-test testPriceFeedVersionIsAccurate`.

**Comment by panditdhamdhere** [comment]
Hello @ArnaudBand thank you very much it worked @EngrPips it does not working but thank you very much man

**Reply by PhasedNorth** [reply, marked as answer]
--mt worked fine 
--match-test  the aliases is --mt 

In order to read the help file you can create a file.

At the command prompt type 

forge test --help >  forgeHelp.txt ,  this would create a file called forgeHelp.txt in the root of foundry-fund-me-f23 where you ca.n review when needed

**Reply by EngrPips** [reply]
Interesting, @panditdhamdhere. You have resolved this issue already. Right?

**Reply by panditdhamdhere** [reply]
yes @EngrPips it resolved and thank you @PhasedNorth really helpful
